packages/web/src/pages/explore-page/components/desktop/MoodGrid.tsx

```diff
@@ -24,51 +29,56 @@ export const MoodGrid = () => {
 
   return (
     <Flex
+      ref={ref}
       direction='column'
       mh='l'
       gap={isMobile ? 'l' : 'xl'}
       alignItems='center'
     >
-      <Text
-        variant={isMobile ? 'title' : 'heading'}
-        size={isMobile ? 'l' : 'm'}
-      >
-        {messages.exploreByMood(
-          category === 'all' ? undefined : labelByCategoryView[category]
-        )}
-      </Text>
-      <Flex
-        gap={isMobile ? 'm' : 's'}
-        justifyContent='center'
-        alignItems='flex-start'
-        wrap='wrap'
-      >
-        {Object.entries(MOODS)
-          .sort()
-          .map(([mood, moodInfo]) => (
-            <Paper
-              key={mood}
-              pv='l'
-              ph='xl'
-              gap='s'
-              borderRadius='m'
-              border='default'
-              backgroundColor='white'
-              onClick={() => handleMoodPress(moodInfo.value)}
-              css={{
-                ':hover': {
-                  background: color.neutral.n25,
-                  border: `1px solid ${color.neutral.n150}`
-                }
-              }}
-            >
-              {moodInfo.icon}
-              <Text variant='title' size='s'>
-                {moodInfo.label}
-              </Text>
-            </Paper>
-          ))}
-      </Flex>
+      {!inView ? null : (
```

Comment:
so does the default height of this part get covered by useDeferredElement, or should we put a box here with a default height?